webapi/src/main/scala/org/knora/webapi/util/PermissionUtilV1.scala

```diff
@@ -194,7 +193,7 @@ object PermissionUtilV1 {
     def getUserPermissionV1(subjectIri: IRI,
```

Comment:
Yes, as the Scaladoc says, `getUserPermissionV1` checks permissions on any subject (which could be a resource or a value), while  `getUserPermissionV1WithValueProps` checks permissions on a value, for which the caller happens to have a `ValueProps`.

If want to check permissions on a value, the choice between these two methods depends on the format that you have the data in.